packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -276,6 +276,7 @@ export function createRococo (t: TFunction): EndpointOption {
       },
       {
         info: 'rococoLoomNetwork',
+        isDisabled: true, // https://github.com/polkadot-js/apps/issues/6181
```

Comment:
```suggestion
        isDisabled: true, // Rococo reset
```

As per https://github.com/polkadot-js/apps/pull/6182